install/bin/alarmdelete.bat

```diff
@@ -1 +1 @@
-@$INSTALL_PATH\bin\decj decodes.tsdb.alarm.AlarmDelete %*%
+@"%~dp0\decj" decodes.tsdb.alarm.AlarmDelete %*%
```

Comment:
I'm not familiar with `%~dp0` is that just part of the windows shell or something that izpack inserts/replaces?

Reply:
That is built into windows batch files.

Reply:
I would like to do something similar to the Linux scripts such as:

```
SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script

Reply:
The installer builder, izpack, is using the opendcs-install.xml to decide what files to install.  

However, for batch files (.bat - windows) every batch file is installed regardless off what components are selected during the install.
This is because of this part of the config:

```xml
	<fileset dir="bin" targetdir="$INSTALL_PATH/bin" defaultexcludes="yes" 
				includes="*.exe,*.sql,*.bat,opendcs.jar" os="windows">
	</fileset>
```			

If this super-simple setup desired, over a hundred lines can be deleted in the rest of the opendcs-install.xml since they have no effect -- especially since I'm updating the batch files so they don't need any parsing during the install process.

Reply:
We should probably go in the other direction and make sure those are properly part of a pack.